src/util/train_types.py

```diff
@@ -0,0 +1,91 @@
+###########################################################
+## types.py
+## 
+## defines
+## - collection of features
+## - feature groups
+## - power labels
+##
+###########################################################
+
+import enum
+import random
+
+SYSTEM_FEATURES = ["nodeInfo", "cpu_scaling_frequency_hertz"]
+
+COUNTER_FEAUTRES = ["cache_miss", "cpu_cycles", "cpu_instructions"]
+CGROUP_FEATURES = ["cgroupfs_cpu_usage_us", "cgroupfs_memory_usage_bytes", "cgroupfs_system_cpu_usage_us", "cgroupfs_user_cpu_usage_us"]
+BPF_FEATURES = ["bpf_cpu_time_us", "bpf_block_irq", "bpf_net_rx_irq", "bpf_net_tx_irq"]
```

Comment:
note that this file differs a bit from the recently pushed train_types.py (since this file is a bit outdated). If possible include the changes from the more recent train_types in a followup PR (or I can include whichever is easier). Some differences include the inclusion of Sole_IRQ_Features since that is going to be included in dyncomp model. Also the inclusion of a LabelGroup (which I think will help greatly in managing our models.)

Reply:
Sorry. I will push the update.

Reply:
@KaiyiLiu1234 I updated only the feature part. Could you please check the rest change related to XGBoost? btw, I think it should not be defined here. 
May related to issue https://github.com/sustainable-computing-io/kepler-model-server/issues/89